### PR TITLE
agentscopt translate_v0

### PR DESCRIPTION
**Pull request by rabZhang**
---
name: Pull Request
about: Create a pull request
---
## Description
添加国际化翻译 引入jquery i18n 文件。  添加持久化状态栏 用于全局切换翻译（悬浮球已做成插件 可以引入其他持久化插件）
[Please describe the background, purpose, changes made, and how to test this PR]

## Checklist

Please check the following items before code is ready to be reviewed.

- [ ]  Code has passed all tests
- [ ]  Docstrings have been added/updated in Google Style
- [ ]  Documentation has been updated
- [ ]  Code is ready for review

### CODE REVIEW

File: src/agentscope/studio/static/i18n/i18n_cn.json [hunk 1]
```diff
@@ -0,0 +1,178 @@
+{
+    "index.Tutorial": "教程",
+    "index.API": "API 文档",
+    "index-guide.introduction": "AgentScope Studio 是一个基于 Web 的平台，用于在 AgentScope 中监控、管理和开发多代理应用程序！请随意探索该平台及其功能",
```
Comment (qbc2016) [inline src/agentscope/studio/static/i18n/i18n_cn.json]: please change "代理" to "智能体"

File: src/agentscope/studio/static/i18n/i18n_cn.json [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    "index-guide.WorkstationDetail": "通过拖动来开发您的多代理应用程序。",
+    "index-guide.GalleryDetail": "即将推出 ...",
+    "index-guide.ServerDetail": "管理您的代理服务器（开发中）。",
+    "market-title": "即将推出 ...",
+    "dashboard-run-placeholder-Search":"查询",
+    "dialogue-detail-dialogue-Nodata":"无可用消息",
+    "dialogue-detail-dialogue-User":"用户",
+    "dialogue-detail-dialogue-textarea": "在此输入消息...",
+    "dialogue-detail-dialogue-send-btn": "发送",
+    "workstation-Example":"事例",
```
Comment (qbc2016) [inline src/agentscope/studio/static/i18n/i18n_cn.json]: "事例" -> "示例"

File: src/agentscope/studio/static/i18n/i18n_cn.json [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    "pipeline-forlooppipeline-BreakFunction": "中断函数",
+    "pipeline-ifelsepipeline-readme": "用 if-else 逻辑实现控制流的模板管道",
+    "pipeline-ifelsepipeline-Condition": "条件函数",
+    "pipeline-msghub-readme": "MsgHub 用于在一组代理之间共享信息",
+    "pipeline-msghub-AnnouncementName": "公告名称",
+    "pipeline-msghub-AnnouncementName-placeholder": "host",
+    "pipeline-msghub-AnnouncementContent": "公告内容",
+    "pipeline-msghub-AnnouncementContent-placeholder": "欢迎加入群聊!",
+    "pipeline-placeholder-readme":"不执行任何操作的占位符",
+    "pipeline-sequentialpipeline-readme": "实现顺序逻辑的模板管道（<b>从上到下</b）",
```
Comment (qbc2016) [inline src/agentscope/studio/static/i18n/i18n_cn.json]: typo  "</b"

File: src/agentscope/studio/static/i18n/i18n_cn.json [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+    "agent-texttoimageagent-readme": "文本到图像生成代理",
+    "agent-texttoimageagent-labelName": "名称",
+    "agent-texttoimageagent-labelName-Assistant": "助理",
+    "agent-texttoimageagent-ModelConfigName": "配置型号名称",
+    "agent-useragent-Copy": "复制",
+    "agent-useragent-readme": "用户的代理服务器",
+    "agent-useragent-labelName": "名称",
+    "agent-useragent-labelName-placeholder": "用户",
+    "message-msg-labelName": "名称",
+    "message-msg-labelName-placeholder": "主机",
```
Comment (qbc2016) [inline src/agentscope/studio/static/i18n/i18n_cn.json]: "主机" -> "主持人"

File: src/agentscope/studio/static/i18n/i18n_cn.json [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+    "workstation-sidebar-item-Tool": "工具",
+    "workstation-surveyContent": "我们希望听到您的意见",
+    "workstation-surveyContent-Button": "参与问卷调查",
+    "agent-dialogagent-Copy": "复制",
+    "agent-dialogagent-readme": "可与用户或其他代理互动的对话代理。",
+    "agent-dialogagent-labelName": "名称",
+    "agent-dialogagent-labelName-Assistant": "助理",
+    "agent-dialogagent-SystemPrompt": "系统提示",
+    "agent-dialogagent-textareaInput-placeholder": "你是助理。",
+    "agent-dialogagent-ModelConfigName": "配置型号名称",
```
Comment (qbc2016) [inline src/agentscope/studio/static/i18n/i18n_cn.json]: "配置型号名称" -> "模型配置名称"

File: src/agentscope/studio/static/i18n/i18n_cn.json [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+    "pipeline-switchpipeline-AddCase": "添加案例",
+    "pipeline-switchpipeline-RemoveCase": "删除案例",
+    "pipeline-whilelooppipeline-readme": "用于实现 while 循环等控制流的模板管道",
+    "pipeline-whilelooppipeline-Condition": "条件函数",
+    "service-bing-search-readme": "在 ReActAgent 中集成必应搜索服务，以增强代理功能",
+    "service-bing-search-Results": "结果编号",
+    "service-execute-python-readme": "在 ReActAgent中集成 Python 解释器，以增强代理功能",
+    "service-google-search-readme": "将 Google 搜索服务集成到 ReActAgent 中以增强代理功能",
+    "service-read-text-readme": "将阅读文本服务集成到 ReActAgent 中以增强代理功能",
+    "service-write-text-readme": "将写入文本服务集成到 ReActAgent 中提升代理能力",
```
Comment (qbc2016) [inline src/agentscope/studio/static/i18n/i18n_cn.json]: There are more services and tools in workstation, please add translations for them.

File: src/agentscope/studio/static/i18n/i18n_cn.json [hunk 7]
```diff
@@ -0,0 +1,206 @@
+{
+    "index.Tutorial": "教程",
+    "index.API": "API 文档",
+    "index-guide.introduction": "AgentScope Studio 是一个基于 Web 的平台，用于在 AgentScope 中监控、管理和开发多智能体应用程序！请随意探索该平台及其功能",
+    "index-guide.h2": "开始探索 AgentScope Studio",
+    "index-guide.DashboardDetail": "监控和管理您的 AgentScope 运行实例.",
+    "index-guide.WorkstationDetail": "通过拖动来开发您的多代理应用程序。",
```
Comment (qbc2016) [inline src/agentscope/studio/static/i18n/i18n_cn.json]: "代理" -> "智能体"，下面还有很多，请搜索替换一下

File: src/agentscope/studio/static/i18n/i18n_cn.json [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+    "index-guide.ServerDetail": "管理您的代理服务器（开发中）。",
+    "market-title": "即将推出 ...",
+    "dashboard-run-placeholder-Search":"查询",
+    "dialogue-detail-dialogue-Nodata":"无可用消息",
+    "dialogue-detail-dialogue-User":"用户",
+    "dialogue-detail-dialogue-textarea": "在此输入消息...",
+    "dialogue-detail-dialogue-send-btn": "发送",
+    "workstation-Example":"示例",
+    "workstation-sidebar-subitem-Agents": "两个代理",
+    "workstation-sidebar-subitem-Pipeline": "管道",
```
Comment (qbc2016) [inline src/agentscope/studio/static/i18n/i18n_cn.json]: "管道" -> "流程" or "基础流程"?  cc @rayrayraykk

File: src/agentscope/studio/static/i18n/i18n_cn.json [hunk 9]
```diff
@@ -0,0 +1,206 @@
+{
+    "index.Tutorial": "教程",
+    "index.API": "API 文档",
+    "index-guide.introduction": "AgentScope Studio 是一个基于 Web 的平台，用于在 AgentScope 中监控、管理和开发多智能体应用程序！请随意探索该平台及其功能",
```
Comment (qbc2016) [inline src/agentscope/studio/static/i18n/i18n_cn.json]: 该页面显示的文字，标点符号格式请保持一致。

**Review by AgentScopeTeam — APPROVED** [review]

**Review by rayrayraykk — APPROVED** [review]

**Pull request merged**